src/components/templates/Jazzbar/JazzBar/JazzBar.tsx

```diff
@@ -1,161 +1,62 @@
-import { VenueTemplate } from "types/VenueTemplate";
 
-import { convertToEmbeddableUrl } from "utils/embeddableUrl";
 import { WithId } from "utils/id";
 
 import { useAnalytics } from "hooks/useAnalytics";
-import { useExperience } from "hooks/useExperience";
 import { useRelatedVenues } from "hooks/useRelatedVenues";
-import { useShowHide } from "hooks/useShowHide";
-import { useUpdateTableRecentSeatedUsers } from "hooks/useUpdateRecentSeatedUsers";
 import { useUser } from "hooks/useUser";
 
 import { RenderMarkdown } from "components/organisms/RenderMarkdown";
 
+import { Loading } from "components/molecules/Loading";
 import { TableComponent } from "components/molecules/TableComponent";
-import { TablesUserList } from "components/molecules/TablesUserList";
+import { TableGrid } from "components/molecules/TableGrid";
 
 import { BackButton } from "components/atoms/BackButton";
 
-import "./JazzBar.scss";
+import styles from "./JazzBar.module.scss";
 
 interface JazzProps {
   venue: WithId<JazzbarVenue>;
 }
 
 export const JazzBar: React.FC<JazzProps> = ({ venue }) => {
-  const { isShown: showOnlyAvailableTables } = useShowHide();
   const { parentVenue } = useRelatedVenues({ currentVenueId: venue.id });
-  const embedIframeUrl = convertToEmbeddableUrl({
-    url: venue.iframeUrl,
-    autoPlay: venue.autoPlay,
-  });
   const analytics = useAnalytics({ venue });
 
   useBackgroundGradient();
 
-  // @debt what does it do anyway? cache? no useSelector here so probably connected to useUpdateTableRecentSeatedUsers
-  useExperience(venue.name);
-
   const jazzbarTables = venue.config?.tables ?? JAZZBAR_TABLES;
 
-  const [seatedAtTable, setSeatedAtTable] = useState<string>();
   const { userId } = useUser();
 
-  useUpdateTableRecentSeatedUsers(
-    VenueTemplate.jazzbar,
-    seatedAtTable && venue?.id
-  );
-
-  const isReactionsMuted = venue.isReactionsMuted ?? DEFAULT_REACTIONS_MUTED;
-
-  const { hide: enableUserAudio, show: disableUserAudio } = useShowHide(
-    isReactionsMuted
-  );
-
-  useEffect(() => {
-    if (isReactionsMuted) {
-      disableUserAudio();
-    } else {
-      enableUserAudio();
-    }
-  }, [isReactionsMuted, disableUserAudio, enableUserAudio]);
-
   useEffect(() => {
     analytics.trackEnterJazzBarEvent();
   }, [analytics]);
 
-  useEffect(() => {
-    seatedAtTable && analytics.trackSelectTableEvent();
-  }, [analytics, seatedAtTable]);
-
-  const { joinHuddle, inHuddle } = useVideoHuddle();
-
-  const joinTable = useCallback(
-    (table) => {
-      // @debt TOOD Fix this
-      joinHuddle(userId || "", `${venue.id}-${table}`);
-      setSeatedAtTable(table);
-    },
-    [joinHuddle, userId, venue.id]
-  );
-
-  const leaveTable = useCallback(async () => {
-    if (!userId) return;
-
-    await unsetTableSeat(userId, { venueId: venue.id });
-    setSeatedAtTable(undefined);
-  }, [userId, venue.id]);
-
-  if (!inHuddle && seatedAtTable) {
-    leaveTable();
+  if (!userId) {
+    return <Loading />;
   }
```

Comment:
Not sure if there isn't a user logged in (i.e. there isn't `userId`), this should display a loading component. A proper check would be `isUserLoading`